core/modules/rproc/InfileMerger.cc

```diff
@@ -284,7 +286,11 @@ bool InfileMerger::finalize() {
         std::string sqlDropCol = std::string("ALTER TABLE ") + _mergeTable
                                + " DROP COLUMN " +  _jobIdColName;
         LOGS(_log, LOG_LVL_DEBUG, "Removing w/" << sqlDropCol);
+        auto begin = std::chrono::system_clock::now();
         finalizeOk = _applySqlLocal(sqlDropCol);
+        auto end = std::chrono::system_clock::now();
+        auto diff = std::chrono::duration_cast<std::chrono::microseconds>(end - begin);
```

Comment:
I've noticed that the performance of some calls made to _applySqlLocal(*) within the compilation unit are measured in this one, meanwhile some others aren't. Wouldn't it make a sense to put this performance measurement (And diagnostic printout) into the method itself to avoid polluting surrounding code?